src/ado2gh/Handlers/AddTeamToRepoCommandHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
```

Comment:
use file scoped namespaces to reduce indentation.

Reply:
good catch, I'll fix them all up

Reply:
I wonder if there's an .editorconfig setting we can turn on for this so linters catch it